PetaPoco/Core/DateTime2Extentions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace PetaPoco
+{
+    public static class DateTime2Extentions
+    {
+        /// <summary>
+        /// Converts an object to its <see cref="DateTime2"/> representation
+        /// </summary>
+        /// <param name="o">The object to be converted</param>
+        /// <returns></returns>
+        public static DateTime2 ToDateTime2(this object o) => new DateTime2((DateTime)o);
```

Comment:
dont like this, every country has its own format

Reply:
It doesn't need to be format specific. You could do something like:

```
public static DateTime2 (this string input) => new DateTime2(DateTime.Parse(input));
```

My point was that I think you only want to be able to pass in a `DateTime` or a string.

Reply:
added in new commit

Reply:
@roggmaeh Thanks, but I still think you want to change the method that takes an object so that it only takes a `DateTime`. Right now, you've got

```c#
public static DateTime2 ToDateTime2(this object o) => new DateTime2((DateTime)o);
```

If do `object foo = 4; foo.ToDateTime2();`, which is perfectly legal code, I get an `InvalidCastException`. I don't think there's a point to offering this method as an extension on all objects, when we know that it only works if the object is a `DateTime.`

Reply:
added in new commit